app/src/main/res/layout/activity_login.xml

```diff
@@ -0,0 +1,99 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".LoginActivity">
+
+    <TextView
+        android:id="@+id/appName"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="100dp"
+        android:layout_marginStart="0dp"
+        android:layout_marginEnd="0dp"
+        android:text="@string/app_name"
+        android:textSize="30sp"
```

Comment:
Use predefined material styles like style="@style/TextAppearance.MaterialComponents.Headline5" to replace the textSize attribute. It will handle both textsize and font weight. Apply that to the loginText Below as well

Reply:
You can use Our Logo (defined in drawables already) in place of the appName text